src/backend/web/routes/feeds.js

```diff
@@ -0,0 +1,53 @@
+const express = require('express');
+const Feed = require('../../data/feed');
+const { getFeeds } = require('../../utils/storage');
+const { logger } = require('../../utils/logger');
+
+const feeds = express.Router();
+
+feeds.get('/', async (req, res) => {
+  let ids;
+  try {
+    ids = await getFeeds();
+  } catch (err) {
+    logger.error({ err }, 'Unable to get feeds from Redis');
+    res.status(503).json({
+      message: 'Unable to connect to database',
+    });
+    return;
+  }
+
```

Comment:
Let's add an `X-Total-Count` header with `ids.length` similar to https://github.com/Seneca-CDOT/telescope/blob/master/src/backend/web/routes/posts.js#L57.